src/components/pages/login/LoginForm.jsx

```diff
@@ -32,7 +38,9 @@ export default function LoginForm() {
         required
         />
         </label>
-      <button>Accéder à votre espace</button>
+      <button onClick={handleOrder}>Accéder à votre espace</button>
```

Comment:
```suggestion
      <button>Accéder à votre espace</button>
```

Pas besoin de onClick ici, tu as déjà onSubmit et handleSubmit